control_server.go

```diff
@@ -86,6 +87,10 @@ func (this *ControlServer) Initialize() (err error) {
 
 	this.router = mux.NewRouter()
 	this.router.HandleFunc("/", this.HandleIndex).Methods("GET")
+	this.router.HandleFunc("/api/progress", this.HandleProgress).Headers("Content-Type", "application/json").Methods("GET")
+	this.router.HandleFunc("/api/cutover", this.HandleCutover).Headers("Content-Type", "application/json").Methods("POST")
+	this.router.HandleFunc("/api/verify", this.HandleVerify).Headers("Content-Type", "application/json").Methods("POST")
+	this.router.HandleFunc("/api/custom_script", this.HandleScript).Headers("Content-Type", "application/json").Methods("POST")
```

Comment:
I see the previous routes are `/api/actions/cutover`, worth it to unify the routes?

Reply:
I think so